distribution/assembly.xml

```diff
@@ -7,6 +7,7 @@
     <formats>
         <format>zip</format>
     </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
```

Comment:
IMO, including the base directory is quite important. Otherwise, unpacking will lead to cumbersome directory structure..